package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack"
+  },
```

Comment:
[OPTIONAL] I would suggest adding the `"start"` script in order to run your app with `npm run start`